feat: [Week-9] Translation and Update of NodeJS Module (#1338)
rolling-scopes-school/tasks: stage1/modules/node-materials/node/projects/notes.md

}
```

> Since the `notes.json` file is in the same directory as the code file, there is no need to specify the path; it's enough to provide its name as the first argument to the `readFile()` method. The second argument of the method is a callback function that takes two parameters - an error (`error`) and the data read from the file (`data`).

Reviewer: ```suggestion
Since the `notes.json` file is located in the same directory as the code file, there's no need to specify its full path. It's enough to provide its name as the first argument to the `readFile()` method. The second argument of this method is a callback function, which accepts two parameters: an `error` for any encountered errors, and the `data` read from the file.
```

Converting the received data into an array is done using the `JSON.parse(data)` method. We iterate through this array using the `forEach()` method and, for each element, print its index and the title of the note to the console, creating a numbered list. To start the list from one instead of zero, we add 1 to the index.

> 6. Write the `view()` function, which outputs the content of a note by its title

Reviewer: ```suggestion
6. Develop the `view()` function, which displays the content of a note based on its title.
```

}
```

> 7. Write the `remove()` function, which deletes a note by its title

Reviewer: ```suggestion
7. Develop the `remove()` function, which deletes a note by its title
```
